en/test_and_ci/integration_testing_mavsdk.md

```diff
@@ -33,4 +33,43 @@ To see all possible command line arguments, check out:
 
 ```sh
 test/mavsdk_tests/mavsdk_test_runner.py -h
+
+usage: mavsdk_test_runner.py [-h] [--log-dir LOG_DIR] [--speed-factor SPEED_FACTOR] [--iterations ITERATIONS] [--abort-early] [--gui] [--model MODEL]
+                             [--case CASE] [--debugger DEBUGGER] [--verbose]
+                             config_file
+
+positional arguments:
+  config_file           JSON config file to use
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --log-dir LOG_DIR     Directory for log files
+  --speed-factor SPEED_FACTOR
+                        How fast to run the simulation
+  --iterations ITERATIONS
+                        How often to run all tests
+  --abort-early         Abort on first unsuccessful test
+  --gui                 Display gzclient with simulation
```

Comment:
FYI this might be useful, but I don't know what gzclient does.

Reply:
Good point. I'll have to fix that in Firmware first and then copy it over.